iib/workers/dogpile_cache.py

```diff
@@ -0,0 +1,64 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+import functools
+
+from dogpile.cache import make_region
+
+from iib.workers.config import get_worker_config
+
+
+def skopeo_inspect_should_use_cache(*args):
```

Comment:
I think it'd be good to consistently have the `should_use_cache` functions to accepts all arguments that were passed in the function call even though it doesn't matter in the case of `skopeo_inspect`.
```suggestion
def skopeo_inspect_should_use_cache(*args, **kwargs):
```